nixos/modules/services/wayland/cage.nix

```diff
@@ -83,6 +83,7 @@ in {
       account required pam_unix.so
       session required pam_unix.so
       session required ${pkgs.systemd}/lib/security/pam_systemd.so
+      session required pam_env.so conffile=${config.system.build.pamEnvironment} readenv=0
```

Comment:
Pleae move it before the `pam_systemd.so` line, to make it consistent with `{sddm,lightdm,gdm}.nix`